gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetFinder.java

```diff
@@ -47,6 +49,8 @@ public class IcebergDatasetFinder implements IterableDatasetFinder<IcebergDatase
   public static final String ICEBERG_HIVE_CATALOG_METASTORE_URI_KEY = ICEBERG_DATASET_PREFIX + ".hive.metastore.uri";
   public static final String ICEBERG_DB_NAME = ICEBERG_DATASET_PREFIX + ".database.name";
   public static final String ICEBERG_TABLE_NAME = ICEBERG_DATASET_PREFIX + ".table.name";
+  public static final String ICEBERG_CATALOG_SPECIFIER_CLASS_KEY = DatasetConstants.PLATFORM_ICEBERG + "catalog.specifier.class";
```

Comment:
let's disambiguate that this is for the 'source' catalog.  there may be a different one for the destination side of the copy

Reply:
gotcha!